<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-VOLT5L75" failures="1" tests="2" name="Default test(failed)" time="0.09" errors="0" timestamp="2020-01-07T00:13:41 IST">
  <testcase classname="TestNG.TestNG_FailedTestCase" name="test3" time="0.013">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestNG.TestNG_FailedTestCase.test3(TestNG_FailedTestCase.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="TestNG.TestNG_FailedTestCase" name="test2" time="0.002"/>
  <testcase classname="TestNG.TestNG_FailedTestCase" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
